tests/containers/single_container_webui.pm

```diff
@@ -3,16 +3,39 @@ use testapi;
 use utils;
 
 sub run {
+  script_run(
+        "echo  \"\$(cat <<EOF
+[openqa_webui]
+key = 1234567890ABCDEF
+secret = 1234567890ABCDEF
+EOF
+)\" > /root/openQA/container/webui/conf/client.conf");
+  script_run(
+        "echo  \"\$(cat <<EOF
+[global]
+# change to the URL the web UI will be available under so redirection for
+# authentication works
+base_url = http://openqa_webui
+
+[auth]
+method = Fake
```

Comment:
https://github.com/os-autoinst/openQA/pull/6766 should help you to not need to put this into the config file.

Reply:
How is this MR supposed to help here? What would be the value to set `-m` to? Reading https://open.qa/docs/#_fake it doesn't mention any other method to enable fake authentication. Or is something like https://github.com/os-autoinst/openQA/blob/05b96119875a37729e11d143ebbc6a402c96d50c/lib/OpenQA/Schema.pm#L39 also implemented in the auth-backend?

Reply:
By setting the mode to "development" or "test" the fake authentication *and* log=debug is automatically enabled in https://github.com/os-autoinst/openQA/blob/05b96119875a37729e11d143ebbc6a402c96d50c/lib/OpenQA/Setup.pm#L270 . So instead of writing a config file one just needs to set the environment variable `OPENQA_WEBUI_MODE=test` when starting the openQA web UI container